### PR TITLE
Update shortlinks

### PR DESCRIPTION
**Pull request by benhylau**
I am removing these:
```
my-issues,https://github.com/issues?q=org:hyphacoop+sort:updated-desc+involves:YOU
agendas,https://github.com/hyphacoop/december-retreat
contacts,https://docs.google.com/spreadsheets/d/1LyElXvLj7ASclSPQqcgOnuEWqBWBStilEfOhIosZzhA/edit#gid=1538436479
finances,https://docs.google.com/spreadsheets/d/1ZHCGpTCA3YNvYI4aW1dU9UKsCeMyAcR5oPmB2yAavOs/edit
```

### CODE REVIEW

File: shortlinks.csv [hunk 1]
```diff
@@ -32,36 +30,35 @@ logs,https://circleci.com/gh/hyphacoop/worker-coop-scripts/tree/master
...
 biz-wg,https://loomio.hypha.coop/g/ojZI2bPl/working-groups-business-planning
 fin-wg,https://loomio.hypha.coop/g/sRPwaorg/working-groups-finance
-gov-wg,https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance-by-laws-incorporation-articles-gm-
-ops-wg,https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
-inf-wg,https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
-finances,https://docs.google.com/spreadsheets/d/1ZHCGpTCA3YNvYI4aW1dU9UKsCeMyAcR5oPmB2yAavOs/edit
+gov-wg,https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance
+ops-wg,https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure
+inf-wg,https://loomio.hypha.coop/g/o5Q0pZKS/working-groups-operations
+#finances,https://docs.google.com/spreadsheets/d/1ZHCGpTCA3YNvYI4aW1dU9UKsCeMyAcR5oPmB2yAavOs/edit
```
Comment (benhylau) [inline shortlinks.csv]:
```suggestion
```

File: shortlinks.csv [hunk 2]
```diff
@@ -71,3 +67,6 @@ payroll,https://docs.google.com/spreadsheets/d/1sfWOQKogtGU83eIIVD0GQnFvRyJzaqzW
 wage,https://docs.google.com/spreadsheets/d/1sfWOQKogtGU83eIIVD0GQnFvRyJzaqzWvrtB3D0jy64/edit#gid=977274894
 projection,https://docs.google.com/spreadsheets/d/1sfWOQKogtGU83eIIVD0GQnFvRyJzaqzWvrtB3D0jy64/edit#gid=0
 okrs,https://docs.google.com/spreadsheets/d/1hMFS3IhzZOFQA-yjt7lFcoVO0Ry6erFEH7D1NswvDtA/edit
+template,https://raw.githubusercontent.com/hyphacoop/organizing/master/-meeting-template.md
+template-standup,https://raw.githubusercontent.com/hyphacoop/organizing/master/-standup-template.md
+template-wg,https://raw.githubusercontent.com/hyphacoop/organizing/master/-wg-template.md
```
Comment (benhylau) [inline shortlinks.csv]:
```suggestion
standup-template,https://raw.githubusercontent.com/hyphacoop/organizing/master/-standup-template.md
wg-template,https://raw.githubusercontent.com/hyphacoop/organizing/master/-wg-template.md
```

File: shortlinks.csv [hunk 3]
```diff
@@ -32,36 +30,34 @@ logs,https://circleci.com/gh/hyphacoop/worker-coop-scripts/tree/master
 interviews,https://docs.google.com/spreadsheets/d/1vjnT0afFh-Ss1BCYZg7nuLyjwZqzRLsSzkhej_OdoFo/edit
 schedules,https://www.when2meet.com/?8177574-eGR5V
 donations,https://docs.google.com/spreadsheets/d/1DpnzCpXWgQr66nFJFIlpa5XSBrt8IfZaLVZH0_q6kcA/edit#gid=2079904514
-working-open,https://github.com/hyphacoop/handbook/blob/master/working-open.md
+working-open,https://handbook.hypha.coop/working-open.html
 calls,https://jitsi.hypha.coop/hyphacoop
 calendar,https://calendar.google.com/calendar/embed?src=s2224p8sptnujs736vplf9anjo@group.calendar.google.com
+availability,https://calendar.google.com/calendar/embed?src=e2gv1tt0kkqq59c8jh7tgknvc8@group.calendar.google.com
```
Comment (dcwalk) [inline shortlinks.csv]: Wasn't availability supposed to be different?

**Review by patcon — APPROVED** [review]
Thx! Do it!

**Pull request merged**